<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForbidChess</class>
 <widget class="QDialog" name="ForbidChess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>禁用棋子</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="blackGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>491</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>黑方棋子</string>
   </property>
   <widget class="QCheckBox" name="black_jv_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_ma_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>马</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_xiang_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>相</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_shi_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>士</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_shi_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>士</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_xiang_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>相</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_ma_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>马</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_jv_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_pao_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>炮</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_pao_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>炮</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_bing_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_bing_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_bing_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_bing_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="black_bing_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="redGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>491</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>红方棋子</string>
   </property>
   <widget class="QCheckBox" name="red_jv_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_ma_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>马</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_xiang_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>相</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_shi_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>士</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_shi_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>士</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_xiang_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>相</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_ma_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>马</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_jv_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_pao_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>炮</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_pao_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>炮</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_bing_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_bing_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_bing_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_bing_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="red_bing_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
